src/blog/2023/09/making-sense-of-your-industrial-data.md

```diff
@@ -0,0 +1,85 @@
+
+
+### Node-RED and node-red-contrib-buffer-parser to the rescue
+
+Node-RED is an open-source flow-based development tool for visual programming. It's particularly well-suited for IIoT applications because of its versatility and extensive library of nodes. One such node, node-red-contrib-buffer-parser, provides a solution to the legacy data conversion challenge.
+
+This powerful Node-RED module allows you to parse a Buffer of bytes or an Array of integer data (which, by no coincidence, the popular module node-red-contrib-modbus outputs), and convert it into various data types. It can output pretty much any data type, including byte-swapped data, WORD swapped data, masked/shifted/scaled data, and even individual bits.
+
+Here's a quick overview of how it works:
+
+1. **Data Parsing**: Start by setting up a ModBus READ node in Node-RED to retrieve data from your industrial device. Then, use the buffer parser node to parse the ModBus data.
+
+1. **Data Conversion**: With the buffer parser, you can easily convert the 16-bit unsigned data into more meaningful formats. Whether you need to translate it into Float, Long, String, or even extract specific bits, this tool makes the process straightforward.
+
+1. **Publishing to MQTT, influxDB, a dashboard, an IIoT system**: Once your data is in a usable format, Node-RED enables you to publish it to many places. MQTT (Message Queuing Telemetry Transport), a popular protocol for IIoT communication is a perfect example. This makes your data accessible to other IIoT systems and applications for further analysis and action.
+
+### Unlocking the Potential of Legacy Data
+
+By leveraging Node-RED and buffer parser, you can bridge the gap between legacy industrial protocols and the IIoT world. This means you can extract valuable insights from your existing infrastructure without the need for costly hardware upgrades or replacements.
+
+In the era of the Industrial Internet of Things, making sense of your industrial data is no longer a daunting challenge. With the right tools approach, you can unlock the full potential of your legacy data and drive efficiency, productivity, and innovation in your industrial processes.  Yey!
+
+### Learn More
+
+We will be publishing follow-up blog posts with more details, best practices and examples on how to use Node-RED to make sense of your industrial data. In the meantime, you can learn more about these tools by visiting the following links:
+* [Node-RED blog posts](https://flowfuse.com/blog/node-red/)
+* [Node-RED videos](https://www.youtube.com/playlist?list=PLpcyqc7kNgp09XeRx_cae1fEIOloPqM1C)
+* [Buffer Parser Node](https://flows.nodered.org/node/node-red-contrib-buffer-parser) 
+
+### A teaser
```

Comment:
I think this is more than a teaser, and thus propose to move this section to above the 'Learn more' section, and call it "3 quick examples" or so.

Reply:
No problemo. Will do that last (after other parts are resolved.

Reply:
resolved in [6958872](https://github.com/flowforge/website/pull/1111/commits/6958872243434546368aa0dccea78bf37412a0c9)